### PR TITLE
Create Data Structure and Algorithms.yml

### PR DESCRIPTION
**Pull request by kshitizsaini113**
I want my repo to be listed in up-for-grabs

### CODE REVIEW

File: _data/projects/Data Structure and Algorithms.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+desc: Repository for implementing all Data Structre and Algorithms
+site: https://github.com/kshitizsaini113/Data-Structure-and-Algorithms
+tags:
+  - c
+  - c++
+  - data-structure
+  - algorithm
+  - contributions
+upforgrabs:
+  name: Data Structure and Algorithms
```
Comment (shiftkey) [inline _data/projects/Data Structure and Algorithms.yml]: I believe this is what your label should be:

```suggestion
  name: up-for-grabs
```

**Review by shiftkey — CHANGES_REQUESTED** [review]
@kshitizsaini113 thanks for the contribution!

There's one change here to make that should appease the automation for the project.

**Comment by kshitizsaini113** [comment]
I think merging is possible now.

**Comment by ritwik12** [comment]
Lgtm!!

**Comment by kshitizsaini113** [comment]
So when will my changes be merged

**Comment by ritwik12** [comment]
@kshitizsaini113 Thanks!!

**Comment by kshitizsaini113** [comment]
Thank you

**Pull request merged**